.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.18
+        go-version: 1.19
```

Comment:
next time around, we should take it to `1.21`, and put it in quotes, because it will otherwise interpret `1.20` as `1.2`, which has bitten me at least 5 times.